portfolio/src/main/webapp/script.js

```diff
@@ -47,18 +47,101 @@ function toggleSidebar(){
     isSidebarOpen = !isSidebarOpen;
 }
 
+/**
+ * Opens the sidebar.
+ * @param {object} el - Sidebar DOM element.
+ * @param {object} button - Comment header button DOM element.
+ */
 function openSidebar(el, button){
     el.style.width = "0px";
     button.innerText = "Show comments!";
     button.classList.remove('closeButton');
 }
 
+/**
+ * Closes the sidebar.
+ * @param {object} el - Sidebar DOM element.
+ * @param {object} button - Comment header button DOM element.
+ */
 function closeSidebar(el, button){
     el.style.width = "550px";
     button.innerText = "Close";
     button.classList.add('closeButton');
 }
 
+//*********************** SIDEBAR HANDLING ***********************
+/**
+ * Listener to check for invalid inputs in the comment fields
+ */
+function checkForInvalidInputs(){
+    const header = document.getElementById('name-input');
+    const body = document.getElementById('text-input');
+    body.onkeydown = function(e){
+        if (e.key == "Enter"){
+
+            e.preventDefault(); //stops the enter from happening
+
+            const headerInput = header.value;
+            const bodyInput = body.value;
+            
+            let checker = checkInvalids(headerInput, bodyInput, header, body);
+            if (checker === 0) {
+                document.commentForm.submit();
+            }
+        }
+    }
+}
+
+/**
+ * Checks the header and body input fields for invalid inputs
+ * @param {String} headerInput - DOM comment header element.
+ * @param {String} bodyInput - DOM comment body element.
+ * @param {object} header - DOM element of the comment header input.
+ * @param {object} body - DOM element of the comment body input.
+ */
+function checkInvalids(headerInput, bodyInput, header, body){
```

Comment:
consider naming this something more descriptive like "validateComment" so it's more obvious what this function is checking on